doc/source/installation/yum_repo.rst

```diff
@@ -85,7 +92,7 @@ You can install Percona yum repository by running the following commands as a ``
    <tokudb_intro>` and :ref:`MyRocks <myrocks_intro>` storage engines. These
    storage engines are installed as plugins. You can find more information on how
    to install and enable the |TokuDB| storage in the :ref:`tokudb_installation`
-   guide. More information about how to install |MyRocks| can be found in the
```

Comment:
What if to make this call to action in an active voice? 
"Find how to install |MyRocks| in ..."